.github/workflows/release-snapshot.yaml

```diff
@@ -0,0 +1,54 @@
+name: Release SHAPSHOT
+
+on:
+  push:
+    branches:
+      - master
+      - "[0-9]+"
+      - "[0-9]+.[0-9]+"
+
+jobs:
+  release:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Set up JDK 1.8
+        uses: actions/setup-java@v1
+        with:
+          java-version: 1.8
+
+      - name: Cache Gradle packages
+        uses: actions/cache@v2
+        with:
+          path: |
+            ~/.gradle/caches
+            ~/.gradle/wrapper
+          key: ${{ runner.os }}-gradle-${{ hashFiles('**/*.gradle*', '**/gradle-wrapper.properties') }}
+          restore-keys: |
+            ${{ runner.os }}-gradle-
+
+      - name: Set version
+        id: version
+        run: |
+          VERSION=$(gradle :core:properties -q | grep "version:" | awk '{print $2}')
+          echo ::set-output name=version::${VERSION}
+
+      - name: Upload SNAPSHOT versions for scalardb, scalardb-server, scalardb-rpc, and scalardb-schema-loader to Maven Central Repository
+        if: contains(steps.version.outputs.version, '-SNAPSHOT')
```

Comment:
It uploads the SNAPSHOT versions only when the project version contains `-SNAPSHOT`.